package.json

```diff
@@ -3,6 +3,23 @@
   "version": "8.10.1",
   "description": "TypeScript execution environment and REPL for node.js, with source map support",
   "main": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./package": "./package.json",
+    "./package.json": "./package.json",
+    "./dist/bin": "./dist/bin.js",
+    "./dist/bin.js": "./dist/bin.js",
+    "./dist/bin-transpile": "./dist/bin-transpile.js",
+    "./dist/bin-transpile.js": "./dist/bin-transpile.js",
+    "./dist/bin-script": "./dist/bin-script.js",
+    "./dist/bin-script.js": "./dist/bin-script.js",
```

Comment:
Should we provide these as exports or only allow `node node_modules/ts-node/dist/bin` style? Do we want both `.js` and without?

Reply:
I wasn't sure how people are using `ts-node` in the wild, and I couldn't think of any compelling benefits that outweighed the possibility of breaking something.